src/anupSJun20/MissingNumbrinIntegerArray.java

```diff
@@ -0,0 +1,21 @@
+package anupSJun20;
+/*Program 1: Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but one number is missing, 
+you need to write a Java program to find that missing number in an array.*/
+
+public class MissingNumbrinIntegerArray {
+	static int findTheMissingNumber(int[] num,int startNum,int endNum) {
+		int overallSum = (endNum*(endNum+1))/2;
+		int sumofArray = 0;
+		for(int index = 0;index<num.length;index++) {
+			sumofArray = sumofArray+num[index];
+		}
+		return overallSum - sumofArray;
+		
+		
+	}
+	public static void main(String[] args) {
+		int[] num = {1,2,3,4,5,7,10,9,8};
+		System.out.println("The missing number is : " + findTheMissingNumber(num,1,10));
```

Comment:
@AnupSJun20 
Have you tried to make code generic by passing extra arguments in method?
If yes then - 
for ->int[] num = {11,15,12,13} when i call findTheMissingNumber(num,11,15) it's giving me incorrect result.

Reply:
Thanks Aditi for this comment. I believe Maulik said that try and take a hard coded range 1- 10 as of now and he will explain in the class for user defined values and multiple missing numbers. So once Maulik will explain today, will create a new program. 
Yes the code gave you incorrect result as it was created for a hard code value to start from 1.

Reply:
Yes Anup, this will work for hardcoded value 1-10.